web-ng/src/app/components/inline-edit-title/inline-edit-title.component.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Inject, OnDestroy } from '@angular/core';
+import { DataStoreService } from '../../services/data-store/data-store.service';
+import { ProjectService } from '../../services/project/project.service';
+import { Observable, Subject } from 'rxjs';
+import { Project } from '../../shared/models/project.model';
+import { MainPageComponent } from '../main-page/main-page.component';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-inline-edit-title',
+  templateUrl: './inline-edit-title.component.html',
+  styleUrls: ['./inline-edit-title.component.css'],
+})
+export class InlineEditTitleComponent implements OnDestroy {
+  readonly activeProject$: Observable<Project>;
+  lang: string;
+  title!: string;
+  projectId!: string;
+
+  subscriptions: Subscription[] = [];
```

Comment:
Please use a single Subscription instead and call `.add()` to accumulate subscriptions.

Example:
https://github.com/google/ground-platform/blob/101cda202c282ac93467b4c3979eeb59547833d8/web-ng/src/app/components/main-page/main-page.component.ts#L36

Reply:
Fixed it thanks :)